static/js/controllers/edit-user.js

```diff
@@ -247,6 +247,11 @@ var passwordTester = require('simple-password-tester'),
                   $scope.errors.currentPassword = value;
                   $scope.setError();
                 });
+              } else if (err.status === 404) {
+                Translate('online.action.message').then(function(value) {
+                  $scope.errors.currentPassword = value;
+                  $scope.setError();
+                });
```

Comment:
Have you tested this? I just put a breakpoint in a dev instance (beta.dev.medicmobile.org) and then disconnected and it looks like the status code is `-1` not `404`. I think `404` is for when you get to the server but the resource isn't there, so perhaps `-1` is what we need to use here.